app_flutter/lib/main.dart

```diff
@@ -2,11 +2,25 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:async';
+import 'dart:html' as html;
```

Comment:
FYI, if you import dart:html at all you can no longer build for android or iOS. You should attempt to put this behind a plugin interface